release.json

```diff
@@ -8,7 +8,7 @@
     "nightly": {
         "INTEGRATIONS_CORE_VERSION": "master",
         "OMNIBUS_SOFTWARE_VERSION": "0566b54e725fcd66e841579a1d789dd5bb23870f",
-        "OMNIBUS_RUBY_VERSION": "e74158aae17d70d7b7f1d356c1b93589c84a1b0a",
+        "OMNIBUS_RUBY_VERSION": "chouquette/xz_package_extra_files",
```

Comment:
Reminder to revert this file before merging.

Reply:
switched to the newest commit on top of `datadog-5.5.0`